tests/conftest.py

```diff
@@ -0,0 +1,78 @@
+from dataclasses import dataclass
+from typing import Callable, Iterable
+
+import pytest
+from shapely import Point
+
+from koswat.dike.surroundings.point.point_surroundings import PointSurroundings
+
+_point_surroundings_cases = [
+    pytest.param(4, 4, (1.5, 3), id="'A' [0, 4], 'B' [4, 8]"),
+    pytest.param(5, 4, (1.5, 3), id="'A' [0, 5], 'B' [5, 9]"),
+    pytest.param(4, 10, (1.5, 9), id="'A' [0, 4], 'B' [4, 14]"),
+    pytest.param(5, 10, (1.5, 9), id="'A' [0, 5], 'B' [5, 15]"),
+    pytest.param(0, 8, (0, 4.5), id="'B' [0, 8]"),
+    pytest.param(0, 10, (0, 4.5), id="'B' [0, 10]"),
+    pytest.param(0, 12, (0, 10.5), id="'B' [0, 12]"),
+]
+
+
+@dataclass
+class PointSurroundingsTestCase:
```

Comment:
Is it proper to add classes to `conftest.py` rather than fixtures?

Reply:
Of course, or at least I've always seen it this way, it is just another type of resource that in this case is being used complimentary to the fixture below. 
I could have also put the class inside the fixture definition, but for readability I rather "expose" it outside.